tests/cupy_tests/sorting_tests/test_search.py

```diff
@@ -247,73 +247,73 @@ class TestNanArgMin(unittest.TestCase):
     @testing.numpy_cupy_allclose()
     def test_nanargmin_all(self, xp, dtype):
         a = testing.shaped_random((2, 3), xp, dtype)
-        return a._nanargmin()
+        return cupy.nanargmin(a)
```

Comment:
Sorry, `xp.nanargmin(a)` is correct.

Reply:
Okay. so instead of a.*() all test cases now should be xp.*(a)?

Reply:
Yes.

Reply:
Done and pushed.